sycl/test-e2e/ESIMD/esimd_test_utils.hpp

```diff
@@ -8,8 +8,10 @@
 
 #pragma once
 
+#include <sycl/bit_cast.hpp>
 #include <sycl/ext/intel/esimd.hpp>
 #include <sycl/sycl.hpp>
+
```

Comment:
Nit: Remove whitespace

Reply:
I'll remove the other white-space (at L16) now. 

BTW, the define is probably not needed here (windows.h is not included here). It can be removed separately later.